vnext/Microsoft.ReactNative/IReactModuleBuilder.cpp

```diff
@@ -129,6 +129,10 @@ void ReactModuleBuilder::AddSyncMethod(hstring const &name, SyncMethodDelegate c
   m_methods.push_back(std::move(cxxMethod));
 }
 
+void ReactModuleBuilder::AddEventEmitter(hstring const &name, EventEmitterInitializerDelegate const &emitter) noexcept {
```

Comment:
>AddEventEmitter [](http://example.com/codeflow?start=25&length=15)

Why do we need this method then? See the event implementation - we only need the AddInitializer call.

Reply:
Event emitters are only supported on TurboModules.  The ReactModuleBuilder is used when the module is running as a native module.  We could choose to throw and not allow the module to run at all if it's being run as a NativeModule.  I was leaning towards allowing at least the rest of the module to continue to work if its run in that mode.  But I'd be ok making this throw.